src/api/ledger/utils.js

```diff
@@ -29,7 +29,7 @@ function getRecursiveRecur(getter, marker, limit, callback) {
 }
 
 function getRecursive(getter, limit, callback) {
-  getRecursiveRecur(getter, undefined, limit, callback);
+  getRecursiveRecur(getter, undefined, limit || Infinity, callback);
```

Comment:
Wait, so I was confused I think, it won't be NaN because limit can't be passed in as undefined, only Infinity. And else where, the Math.min/Math.max stuff should always work fine with `limit=Infinity` to clamp so `10 <= limit <= 400`